fcrepo-jms/src/main/java/org/fcrepo/jms/headers/DefaultMessageFactory.java

```diff
@@ -130,4 +160,17 @@ public String apply(final Integer type) {
 
     private static final Logger LOGGER = getLogger(DefaultMessageFactory.class);
 
+    private String getPropertiesAsString(final FixityEvent fixityEvent) {
+        final StringBuilder propsStrBuilder = new StringBuilder();
+        for (Map.Entry<String,String> entry : fixityEvent.getInfo().entrySet()) {
+            propsStrBuilder.append(entry.getKey()).append("->").append(entry.getValue()).append(",");
```

Comment:
The current behavior in publishing the `org.fcrepo.jms.properties` header is to include a list of the changed properties. 

This PR represents a departure from that by including the corresponding values, delimited by the "->" string. This will result in a message header like this: 
`http://www.loc.gov/premis/rdf/v1#hasMessageDigest->urn:sha1:b05a0e241d65e3016a364fe9f5105381ea4a7682, http://www.loc.gov/premis/rdf/v1#hasSize->13564^^http://www.w3.org/2001/XMLSchema#long, http://fedora.info/definitions/v4/repository#status->SUCCESS`

(I added spaces after the commas above).

While it is certainly necessary to publish whether the fixity event was successful or not, doing so in this way does not appear to be the best method. At the very least, a better delimiter should be used. Alternately, the success (or failure) should be published in a dedicated header field such as `org.fcrepo.jms.status` or `org.fcrepo.jms.fixity`.